app/assets/stylesheets/views/_guide.scss

```diff
@@ -21,4 +21,12 @@
       @include bold-24;
     }
   }
+
+  .task-list-sidebar {
```

Comment:
See similar comment in guides PR: https://github.com/alphagov/government-frontend/pull/713#discussion_r162672603